### PR TITLE
added account type to topics

### LINKED ISSUE #7: Investigate the data related to poster role

### LINKED ISSUE #20: Implement poster role for topics

### PR DESCRIPTION
**Pull request by connormaas-cmu**
Fixes #20
Fixes #7

### CODE REVIEW

**Comment by TyreceDJ** [comment]
Remove the .github/workflow/hw1.yaml file from your commit as it says in the project write-up. The HW1 tests will fail.

File: src/topics/index.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ Topics.getTopicsByTids = async function (tids, options) {
 
         const [teasers, users, userSettings, categoriesData, guestHandles, thumbs] = await Promise.all([
             Topics.getTeasers(topics, options),
-            user.getUsersFields(uids, ['uid', 'username', 'fullname', 'userslug', 'reputation', 'postcount', 'picture', 'signature', 'banned', 'status']),
+            user.getUsersFields(uids, ['uid', 'username', 'fullname', 'userslug', 'reputation', 'postcount', 'picture', 'signature', 'banned', 'status', "accounttype"]),
```
Comment (TyreceDJ) [inline src/topics/index.js]: Hi Conner, you can fix the lint error in src/topics/index.js on line 95 by using single quotes instead of double.

Comment (connormaas-cmu) [reply src/topics/index.js]: Got it.

**Comment by TyreceDJ** [comment]
Looks good!

**Pull request merged**